support/k8s/k8s-workload-registrar/webhookhandler.go

```diff
@@ -3,25 +3,104 @@ package main
 import (
 	"context"
 	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"net/http"
 
+	admv1 "k8s.io/api/admission/v1"
 	admv1beta1 "k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/klog/v2"
```

Comment:
This project uses `logrus` for logging. You can pass the `logrus.FieldLogger` into `NewWebhookHandler` and store it with the `WebHookHandler`. We shouldn't be uslng klog.